static/metricsgraphics/js/main.js

```diff
@@ -0,0 +1,164 @@
+var theme = 'dark';
+
+(function() {
+    'use strict';
+
+    //set the active pill and section on first load
+    var section = (document.location.hash) ? document.location.hash.slice(1) : 'signature_lookup';
+
+    if (section.indexOf("bugzilla") != -1) {
+      $('#trunk').load('charts/bugzilla/' + section + '.htm', function() {
```

Comment:
I thought it was odd to see '.htm' here when this patch mentions only '.html' everywhere else, but as long as it works, woot.

Reply:
I just pulled in the whole MetricsGraphics.js library I was using for another project. Line 10 above isn't even hit in the context of this change to Bugzilla but it's part of the library so I left it in.